modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/IgnitionTest.java

```diff
@@ -79,8 +78,8 @@
 
     /** */
     @AfterEach
-    void tearDown() throws Exception {
-        IgniteUtils.closeAll(startedNodes);
+    void tearDown() {
+        startedNodes.stream().map(Ignite::name).forEach(IgnitionManager::stop);
```

Comment:
`closeAll` closes all components except those that could not be stopped. `forEach` stops iteration immediately after the first component fils on stop. Is it ok for the test?

Reply:
Fixed.